upgrade/1.4.4/CSM-Only.md

```diff
@@ -0,0 +1,148 @@
+
+**If other products are installed on the system, return to [Update NCN images](../1.4.4/README.md#update-ncn-images) and
+choose option 2.**
+
+## Requirements
+
+* `CSM_RELEASE` is set in the shell environment on `ncn-m001`.
+
+## Steps
+
+1. (`ncn-m001#`) Generate a new CFS configuration for the management nodes.
+
+   This script creates a new CFS configuration that includes the CSM version in its name and applies it to the
+   management nodes. This leaves the management node components in CFS disabled. They will be automatically enabled when
+   they are rebooted at a later stage in the upgrade.
+
+   ```bash
+   /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
+       --no-enable --config-name management-${CSM_RELEASE}
+   ```
+
+   Successful output should end with the following line:
+
+   ```text
+   All components updated successfully.
+   ```
+
+1. Set the IMS IDs for each Management node subrole.
+
+   ```bash
+   export MASTER_IMAGE_ID=""
+   ```
+
+   ```bash
+   export WORKER_IMAGE_ID=""
+   ```
+
+   ```bash
+   export STORAGE_IMAGE_ID=""
+   ```
+
+1. Create new CFS sessions for both Kubernetes and Storage nodes.
+
+    1. Build master images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Master \
+           "$MASTER_IMAGE_ID}" \
+           --target-definition image \
+           --target-image-map "$MASTER_IMAGE_ID" "" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "management-master-${CSM_RELEASE}-upgrade" \
+           --format json
+       ```
+
+    1. Build storage images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Storage \
+           "$STORAGE_IMAGE_ID" \
+           --target-definition image \
+           --target-image-map "$STORAGE_IMAGE_ID" "" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "management-storage-${CSM_RELEASE}-upgrade" \
+           --format json
+       ```
+
+    1. Build worker images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Worker \
+           "$WORKER_IMAGE_ID" \
+           --target-definition image \
+           --target-image-map"$WORKER_IMAGE_ID" "" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "management-worker-${CSM_RELEASE}-upgrade" \
+           --format json
+       ```
+
+1. Wait forever for images to build.
+
+1. Set the new IMS Image IDs
+
+   ```bash
+   export NEW_MASTER_IMAGE_ID=""
+   ```
+
+   ```bash
+   export NEW_WORKER_IMAGE_ID=""
+   ```
+
+   ```bash
+   export NEW_STORAGE_IMAGE_ID=""
+   ```
+
+1. Assign images in BSS
+
+    1. Update master management nodes:
+
+       ```bash
+       /usr/share/doc/csm/scripts/operations/configuration/node_management/assign-ncn-images.sh \
+           -m \
+           -p "$NEW_MASTER_IMAGE_ID"
+       ```
```

Comment:
[Same comment|https://github.com/Cray-HPE/docs-csm/pull/4647/files#r1476874947], but instead of removing this I could just get both Masters and Workers with:

```bash
       /usr/share/doc/csm/scripts/operations/configuration/node_management/assign-ncn-images.sh \
           -mw \
           -p "$NEW_KUBERNETES_IMAGE_ID"
```

Reply:
Resolved, using `-mw`.